compiler/ballerina-parser/src/test/resources/statements/assignment-stmt/assignment_stmt_assert_13.json

```diff
@@ -1,122 +1,90 @@
-            {
-              "kind": "WHITESPACE_MINUTIAE",
-              "value": " "
+              "kind": "PLUS_TOKEN",
+              "trailingMinutiae": [
+                {
+                  "kind": "WHITESPACE_MINUTIAE",
+                  "value": " "
+                }
+              ]
             },
             {
-              "kind": "INVALID_NODE_MINUTIAE",
-              "invalidNode": {
-                "kind": "INVALID_TOKEN_MINUTIAE_NODE",
-                "children": [
-                  {
-                    "kind": "IDENTIFIER_TOKEN",
-                    "value": "b"
-                  }
-                ]
-              }
-            },
-            {
-              "kind": "WHITESPACE_MINUTIAE",
-              "value": " "
-            },
-            {
-              "kind": "INVALID_NODE_MINUTIAE",
-              "invalidNode": {
-                "kind": "INVALID_TOKEN_MINUTIAE_NODE",
-                "children": [
-                  {
-                    "kind": "PLUS_TOKEN"
-                  }
-                ]
-              }
-            },
+              "kind": "SIMPLE_NAME_REFERENCE",
+              "children": [
+                {
+                  "kind": "IDENTIFIER_TOKEN",
+                  "value": "b",
+                  "trailingMinutiae": [
+                    {
+                      "kind": "WHITESPACE_MINUTIAE",
+                      "value": " "
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "kind": "PLUS_TOKEN",
+          "trailingMinutiae": [
             {
               "kind": "WHITESPACE_MINUTIAE",
               "value": " "
-            },
-            {
-              "kind": "INVALID_NODE_MINUTIAE",
-              "invalidNode": {
-                "kind": "INVALID_TOKEN_MINUTIAE_NODE",
-                "children": [
-                  {
-                    "kind": "IDENTIFIER_TOKEN",
-                    "value": "c"
-                  }
-                ]
-              }
-            },
+            }
+          ]
+        },
+        {
+          "kind": "SIMPLE_NAME_REFERENCE",
+          "children": [
             {
-              "kind": "WHITESPACE_MINUTIAE",
-              "value": "  "
+              "kind": "IDENTIFIER_TOKEN",
+              "value": "c",
+              "trailingMinutiae": [
+                {
+                  "kind": "WHITESPACE_MINUTIAE",
+                  "value": "  "
+                }
+              ]
             }
           ]
         }
       ]
     },
     {
-      "kind": "EQUAL_TOKEN",
+      "kind": "SEMICOLON_TOKEN",
       "isMissing": true,
       "hasDiagnostics": true,
       "diagnostics": [
-        "ERROR_MISSING_EQUAL_TOKEN"
+        "ERROR_MISSING_SEMICOLON_TOKEN"
       ]
-    },
-    {
-      "kind": "SIMPLE_NAME_REFERENCE",
-      "children": [
-        {
-          "kind": "IDENTIFIER_TOKEN",
-          "value": "d"
```

Comment:
Looks like `a + b + c  d;` recovers `as a + b + c MISSING[;]` and remaining part not included in the json. Shall we use `testFile()` for this test case?

Reply:
Updated